stripe/res/values/attrs.xml

```diff
@@ -21,4 +21,12 @@
         <attr name="countryAutoCompleteStyle" format="reference" />
         <attr name="countryItemLayout" format="reference" />
     </declare-styleable>
+
+    <declare-styleable name="StripeCardFormView">
+        <attr name="cardFormStyle" format="enum">
+            <enum name="standard" value="0"/>
+            <enum name="borderless" value="1"/>
+        </attr>
+        <attr name="formBackgroundColorStateList" format="reference" />
```

Comment:
Can we remove the `form` prefix from `formBackgroundColorStateList`? Can be in a follow-up PR.